### ISSUE TITLE
Conda environment conflicts

### ISSUE THREAD

**Issue by cuttle-fish-my** [naver/poc#3]
Hi! Thanks for your great work! I'm interested in trying it but encountered a problem when installing dependencies. I followed the instructions in `README.md` and ran the following command:
```
conda env create -f poc.yml
```
However, conda reported that there were several conflicts and spent a bunch of time finding incompatible packages. The full log it too long to display, so I just report the end of it.
```
The following specifications were found to be incompatible with your system:

  - feature:/linux-64::__glibc==2.17=0
  - feature:|@/linux-64::__glibc==2.17=0
  - argon2-cffi-bindings==21.2.0=py310h7f8727e_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - astropy==5.1=py310ha9d4c09_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - bcrypt==3.2.0=py310h5eee18b_1 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - blosc==1.21.3=h6a678d5_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - bottleneck==1.3.5=py310ha9d4c09_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - brotli-bin==1.0.9=h5eee18b_7 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - brotli==1.0.9=h5eee18b_7 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - brotlipy==0.7.0=py310h7f8727e_1002 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - brunsli==0.1=h2531618_0 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']
  - bzip2==1.0.8=h7b6447c_0 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']
  - c-ares==1.19.0=h5eee18b_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - cffi==1.15.1=py310h74dc2b5_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - cfitsio==3.470=hf0d0db6_6 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']
  - charls==2.2.0=h2531618_0 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']
  - contourpy==1.0.5=py310hdb19cb5_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - cryptography==39.0.1=py310h9ce1e76_0 -> libgcc-ng -> __glibc[version='>=2.17']
  - curl==7.88.1=h5eee18b_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - cytoolz==0.12.0=py310h5eee18b_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - daal4py==2023.0.2=py310h3c18c91_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - dal==2023.0.1=hdb19cb5_26647 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - dbus==1.13.18=hb2f20db_0 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']
  - debugpy==1.5.1=py310h295c915_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - expat==2.4.9=h6a678d5_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - ffmpeg==4.3=hf484d3e_0 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']
  - fontconfig==2.14.1=h4c34cd2_2 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - freetype==2.12.1=h4a9f257_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - gensim==4.3.0=py310h1128e8f_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - giflib==5.2.1=h5eee18b_3 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - glib==2.69.1=h4ff587b_1 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - gmp==6.2.1=h295c915_3 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - gmpy2==2.1.2=py310heeb90bb_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - gnutls==3.6.15=he1e5248_0 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']
  - greenlet==2.0.1=py310h6a678d5_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - gst-plugins-base==1.14.1=h6a678d5_1 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - gstreamer==1.14.1=h5eee18b_1 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - h5py==3.7.0=py310he06866b_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - hdf5==1.10.6=hb1b8bf9_0 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']
  - icu==58.2=he6710b0_3 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']
  - imagecodecs==2021.8.26=py310h46e8fbd_2 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - jellyfish==0.9.0=py310h7f8727e_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - jpeg==9e=h5eee18b_1 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - jq==1.6=h27cfd23_1000 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']
  - jxrlib==1.1=h7b6447c_2 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']
  - kiwisolver==1.4.4=py310h6a678d5_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - krb5==1.19.4=h568e23c_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - lame==3.100=h7b6447c_0 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']
  - lazy-object-proxy==1.6.0=py310h7f8727e_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - lcms2==2.12=h3be6417_0 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']
  - lerc==3.0=h295c915_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - libaec==1.0.4=he6710b0_1 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']
  - libbrotlicommon==1.0.9=h5eee18b_7 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - libbrotlidec==1.0.9=h5eee18b_7 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - libbrotlienc==1.0.9=h5eee18b_7 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - libclang13==14.0.6=default_he11475f_1 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - libclang==14.0.6=default_hc6dbbc7_1 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - libcurl==7.88.1=h91b91d3_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - libdeflate==1.17=h5eee18b_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - libedit==3.1.20221030=h5eee18b_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - libev==4.33=h7f8727e_1 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - libevent==2.1.12=h8f2d780_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - libffi==3.3=he6710b0_2 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']
  - libgcc-ng==11.2.0=h1234567_1 -> __glibc[version='>=2.17']
  - libgfortran-ng==7.5.0=ha8ba4b0_17 -> __glibc[version='>=2.17']
  - libiconv==1.16=h7f8727e_2 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - libidn2==2.3.2=h7f8727e_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - libllvm11==11.1.0=h9e868ea_6 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - libllvm14==14.0.6=hef93074_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - libnghttp2==1.46.0=hce63b2e_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - libpng==1.6.39=h5eee18b_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - libpq==12.9=h16c4e8d_3 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - libprotobuf==3.20.3=he621ea3_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - libsodium==1.0.18=h7b6447c_0 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']
  - libspatialindex==1.9.3=h2531618_0 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']
  - libssh2==1.10.0=h8f2d780_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - libstdcxx-ng==11.2.0=h1234567_1 -> __glibc[version='>=2.17']
  - libtasn1==4.19.0=h5eee18b_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - libtiff==4.5.0=h6a678d5_2 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - libunistring==0.9.10=h27cfd23_0 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']
  - libuuid==1.41.5=h5eee18b_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - libwebp-base==1.2.4=h5eee18b_1 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - libwebp==1.2.4=h11a3e52_1 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - libxcb==1.15=h7f8727e_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - libxkbcommon==1.0.1=h5eee18b_1 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - libxml2==2.10.3=hcbfbd50_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - libxslt==1.1.37=h2085143_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - libzopfli==1.0.3=he6710b0_0 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']
  - llvmlite==0.39.1=py310he621ea3_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - lxml==4.9.2=py310h5eee18b_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - lz4-c==1.9.4=h6a678d5_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - lz4==3.1.3=py310h7f8727e_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - lzo==2.10=h7b6447c_2 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']
  - markupsafe==2.1.1=py310h7f8727e_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - matplotlib-base==3.7.1=py310h1128e8f_1 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - mistune==0.8.4=py310h7f8727e_1000 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - mkl-service==2.4.0=py310h7f8727e_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - mkl_fft==1.3.1=py310hd6ae3a3_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - mkl_random==1.2.2=py310h00e6091_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - mpc==1.1.0=h10f8cd9_1 -> libgcc-ng[version='>=7.2.0'] -> __glibc[version='>=2.17']
  - mpfr==4.0.2=hb69a4c5_1 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']
  - mpich==3.3.2=hc856adb_0 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']
  - msgpack-python==1.0.3=py310hd09550d_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - ncurses==6.4=h6a678d5_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - nettle==3.7.3=hbbd107a_1 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']
  - ninja-base==1.10.2=hd09550d_5 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - nspr==4.33=h295c915_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - nss==3.74=h0370c37_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - numba==0.56.4=py310h1128e8f_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - numexpr==2.8.4=py310h8879344_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - numpy-base==1.22.3=py310h9585f30_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - numpy==1.22.3=py310hfa59a62_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - oniguruma==6.9.7.1=h27cfd23_0 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']
  - openh264==2.1.1=h4ff587b_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - openjpeg==2.4.0=h3ad879b_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - openpyxl==3.0.10=py310h5eee18b_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - openssl==1.1.1t=h7f8727e_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - pandas==1.5.3=py310h1128e8f_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - pcre==8.45=h295c915_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - pillow==9.4.0=py310h6a678d5_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - psutil==5.9.0=py310h5eee18b_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - pycurl==7.45.2=py310h37d81fd_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - pyerfa==2.0.0=py310h7f8727e_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - pyodbc==4.0.34=py310h6a678d5_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - pyqt==5.15.7=py310h6a678d5_1 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17|>=2.17,<3.0.a0']
  - pyqtwebengine==5.15.7=py310h6a678d5_1 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17|>=2.17,<3.0.a0']
  - pyrsistent==0.18.0=py310h7f8727e_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - pytables==3.7.0=py310hf19a122_1 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - python-snappy==0.6.1=py310h6a678d5_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - python==3.10.6=haa1d7c7_1 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - pywavelets==1.4.1=py310h5eee18b_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - pyyaml==6.0=py310h5eee18b_1 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - pyzmq==23.2.0=py310h6a678d5_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - qt-main==5.15.2=h8373d8f_8 -> __glibc[version='>=2.17,<3.0.a0']
  - qt-main==5.15.2=h8373d8f_8 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - qt-webengine==5.15.9=hbbf29b9_6 -> __glibc[version='>=2.17,<3.0.a0']
  - qt-webengine==5.15.9=hbbf29b9_6 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - qtwebkit==5.212=h3fafdc1_5 -> __glibc[version='>=2.17,<3.0.a0']
  - qtwebkit==5.212=h3fafdc1_5 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - readline==8.2=h5eee18b_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - regex==2022.7.9=py310h5eee18b_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - scikit-image==0.19.3=py310h6a678d5_1 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - scikit-learn==1.2.2=py310h6a678d5_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - scipy==1.7.3=py310hfa59a62_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - sip==6.6.2=py310h6a678d5_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - snappy==1.1.9=h295c915_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - sqlalchemy==1.4.39=py310h5eee18b_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - sqlite==3.41.2=h5eee18b_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - statsmodels==0.13.5=py310ha9d4c09_1 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - tbb4py==2021.8.0=py310hdb19cb5_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - tbb==2021.8.0=hdb19cb5_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - tk==8.6.12=h1ccaba5_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - tokenizers==0.11.4=py310h3dcd8bd_1 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - tornado==6.2=py310h5eee18b_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - twisted==22.2.0=py310h5eee18b_1 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - ujson==5.4.0=py310h6a678d5_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - unixodbc==2.3.11=h5eee18b_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - wrapt==1.14.1=py310h5eee18b_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - xz==5.2.10=h5eee18b_1 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - yaml==0.2.5=h7b6447c_0 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']
  - zeromq==4.3.4=h2531618_0 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']
  - zfp==0.5.5=h295c915_6 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - zlib==1.2.13=h5eee18b_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']
  - zope.interface==5.4.0=py310h7f8727e_0 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - zstd==1.5.5=hc292b87_0 -> libgcc-ng[version='>=11.2.0'] -> __glibc[version='>=2.17']

Your installed version is: 2.17
```

For your reference, the log of `conda info` is as follows:
```
     active environment : base
    active env location : /public/software/anaconda3
            shell level : 1
       user config file : /public/home/libn/.condarc
 populated config files : /public/home/libn/.condarc
          conda version : 4.10.3
    conda-build version : 3.21.4
         python version : 3.8.8.final.0
       virtual packages : __linux=3.10.0=0
                          __glibc=2.17=0
                          __unix=0=0
                          __archspec=1=x86_64
       base environment : /public/software/anaconda3  (read only)
      conda av data dir : /public/software/anaconda3/etc/conda
  conda av metadata url : None
           channel URLs : https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main/linux-64
                          https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main/noarch
                          https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/r/linux-64
                          https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/r/noarch
                          https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/msys2/linux-64
                          https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/msys2/noarch
          package cache : /public/software/anaconda3/pkgs
                          /public/home/libn/.conda/pkgs
       envs directories : /public/home/libn/.conda/envs
                          /public/software/anaconda3/envs
               platform : linux-64
             user-agent : conda/4.10.3 requests/2.25.1 CPython/3.8.8 Linux/3.10.0-957.el7.x86_64 centos/7.6.1810 glibc/2.17
                UID:GID : 1362:1007
             netrc file : None
           offline mode : False
```
I'll be appreciated if you can help me, thanks in advance!

**Comment by pdejorge** [comment]
Hello @cuttle-fish-my were you able to solve this? I'm afraid I can not be of much help since I can not reproduce the error but in the readme you will find an explanation of how to install the dependencies yourself from the repos that we use (rather than using our readme). Have you tried that?

**Comment by YLBGHXTX** [comment]
@cuttle-fish-my Excuse me, I have the same problem, have you solved it?

**Comment by cuttle-fish-my** [comment]
Hi @YLBGHXTX @pdejorge, I’ve solved the problem by following the installation instruction from mask2former and installing the rest packages manually, which is indeed laborious. 

Note that there is a package called MultiScaleDeformableAttenttion(correct me if I’m wrong) requires CUDA compilation and can’t be directly installed from poc.yaml, so it’s better to follow the official installation instructions from [mask2former](https://github.com/facebookresearch/Mask2Former/blob/main/INSTALL.md).

**Comment by pdejorge** [comment]
Hi @cuttle-fish-my, @YLBGHXTX just to clarify for future reference. Our codebase is built on top of different repos from previous work so we used different conda environments for different parts. I have updated the readme to clarify that, moreover, I have removed the .yml file that was causing conflicts and added instructions to install a conda environment to run the POC inpainting pipeline. 

As @cuttle-fish-my mentioned, the Mask2Former installation requires compiling the MultiScaleDeformableAttenttion and I agree that can be tricky but I would recommend to post issues regarding this in their own [repo](https://github.com/facebookresearch/Mask2Former/blob/main/INSTALL.md)

I'm closing this issue now as I have modified the installation instructions, but if you have more issues feel free to open it again.
